azure_arc_servers_jumpstart/oci/ubuntu/terraform/scripts/vars.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+# Change the following environment variables to your OCI details
+
+export TF_VAR_tenancy_ocid=Oracle tenanacy OCID
+export TF_VAR_user_ocid=Oracle user OCID
+export TF_VAR_fingerprint=Oracle API FingerPrint
+export TF_VAR_private_key_path=oci_api_key.pem
+export TF_VAR_ssh_public_key=$(cat my_oci_key.pub) 
+### OCI Region
+export TF_VAR_region=us-ashburn-1
```

Comment:
Change to description instead of a pre-populated value

Reply:
The instructions have the use building a key called my_oci_key.pem.

Reply:
The region is set to a US region. This is fine for default but consider if users want to deploy to non-US regions.

Reply:
@JerryMSFT I suggest doing something like this for the region:
export TF_VAR_region=<OCI region>

Reply:
modified export TF_VAR_region=us-ashburn-1 to export TF_VAR_region=<"OCI region">